tests/test_salesforce_all_fields_non_custom.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test that with only non-custom fields selected for a stream automatic fields and non custom fields  are still replicated
+"""
+
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from sfbase import SFBaseTest
+
+
+class SFNonCustomFieldsTest(AllFieldsTest, SFBaseTest):
+
+    salesforce_api = 'BULK'
+
+    @staticmethod
+    def name():
+        return "tt_sf_all_fields_non_custom"
+
+    def streams_to_test(self):
+        if self.partitioned_streams:
+            return self.partitioned_streams
+        return self.partition_streams(self.get_streams_with_data())
+
+    def streams_to_selected_fields(self):
+        found_catalogs = AllFieldsTest.found_catalogs
+        conn_id = AllFieldsTest.conn_id
+        non_custom_fields = self.get_non_custom_fields(found_catalogs, conn_id)
+        return non_custom_fields
+
+    def test_non_custom_fields(self):
+        for stream in self.streams_to_selected_fields():
+            expected_non_custom_fields = self.selected_fields.get(stream,set())
+            replicated_non_custom_fields = self.actual_fields.get(stream, set())
+
+            #Verify at least one non-custom field is replicated
+            self.assertIsNotNone(replicated_non_custom_fields,
+                                 msg = f"Replication didn't return any non-custom fields for stream {stream}")
+
+            #verify that all the non_custom fields are replicated
+            self.assertEqual(replicated_non_custom_fields, expected_non_custom_fields,
+                             msg = f"All non_custom fields are not no replicated for stream {stream}")
+
+            #verify that automatic fields are also replicated along with non_custom_fields
+            self.assertTrue(self.expected_automatic_fields(stream).issubset(replicated_non_custom_fields,
```

Comment:
```suggestion
            self.assertTrue(self.expected_automatic_fields(stream).issubset(replicated_non_custom_fields),
```
This missing paren is causing CI to fail